### PR TITLE
Migrate to emersion/go-imap

### LINKED ISSUE #27: Use emersion/go-imap library

### PR DESCRIPTION
**Pull request by emersion**
Fixes #27

### CODE REVIEW

File: backend/imap/conn.go [hunk 1]
```diff
@@ -31,39 +43,34 @@ type conns struct {
...
-		c, err = imap.Dial(b.config.Host())
+		c, err = imapclient.Dial(b.config.Host())
 	}
 	if err != nil {
 		return
 	}
 
-	//c.SetLogMask(imap.LogAll)
-
 	if !b.config.Tls {
```
Comment (akhedrane) [inline backend/imap/conn.go:56]: this should test if TLS is enabled in config.

Comment (emersion) [reply backend/imap/conn.go:56]: What do you mean? This checks if TLS is *not* enabled, and uses STARTTLS if so. Using non-encrypted connections is not secure!

Comment (akhedrane) [reply backend/imap/conn.go:56]: if you forced encrypted connection then TLS config is useless.

Comment (emersion) [reply backend/imap/conn.go:56]: Nope, it's used to distinguish between TLS and STARTTLS connections. Basically, with TLS connections all the traffic is encrypted from the start, but with STARTTLS the connection is unencrypted and a command is used to begin the TLS handshake.

Comment (akhedrane) [reply backend/imap/conn.go:56]: OK, got it
thank you.

**Pull request merged**